components/scream/docs/build.md

```diff
@@ -0,0 +1,93 @@
+# Building and Testing Standalone SCREAM
+
+Follow these simple instructions to build and test SCREAM's standalone
+configuration for yourself. This document makes use of the following
+paths:
+
++ `${RUN_ROOT_DIR}`: the root directory where SCREAM is built and run
++ `${SCREAM_SRC_DIR}`: the directory into which you've cloned the `scream` repo
+
+## 1. Start From a Trustworthy Commit
+
+First, make sure you've cloned the [SCREAM repo](https://github.com/E3SM-Project/scream)
+to `SCREAM_SRC_DIR`. If you're running a branch that's not `master`, check out
+this branch with
+
+```
+git checkout <branch>
+```
+
+## 2. Configure Your SCREAM Build
+
+Create a test directory in `RUN_ROOT_DIR` and use CMake to configure your build.
+This usually looks something like the following:
+```
+mkdir -p $RUN_ROOT_DIR/test
+cmake \
+    -D CMAKE_INSTALL_PREFIX=${RUN_ROOT_DIR}/install \
+    -D CMAKE_BUILD_TYPE=Debug \
+    -D KOKKOS_ENABLE_DEBUG=ON \
+    -D KOKKOS_ENABLE_AGGRESSIVE_VECTORIZATION=OFF \
+    -D KOKKOS_ENABLE_SERIAL=ON \
+    -D KOKKOS_ENABLE_OPENMP=ON \
+    -D KOKKOS_ENABLE_PROFILING=OFF \
+    -D KOKKOS_ENABLE_DEPRECATED_CODE=OFF \
+    -D KOKKOS_ENABLE_EXPLICIT_INSTANTIATION:BOOL=OFF \
+    ${SCREAM_SRC_DIR}
+```
+
+Here, we've configured a `Debug` build to make it easier to find and fix errors.
+For performance testing, you should configure a `Release` build and make use of
+other options, depending on your architecture.
```

Comment:
Should we add a list of possible options? Like `SCREAM_PACK_SIZE` or `SCREAM_FPMODEL` or `SCREAM_FPE`? I don't want to go too much overboard, but we do use some `SCREAM_blah` option in pretty much every build.

Reply:
I like that idea, but maybe that requires more people to chime in. Maybe we can start with this and iterate.

Reply:
I can help.

Reply:
I like the idea of including options. This build.md file is somewhat redundant with https://acme-climate.atlassian.net/wiki/spaces/NGDNA/pages/1264386127/Running+SCREAM+Tests though, which makes me nervous. Perhaps provide a link to this html page in build.md? OR maybe we should only document how to run in confluence (in which case people outside e3sm would never have access to info on how to run?

Reply:
I think the closer to the code the information lives, the more likely it is to be kept up to date. :-) That and your last comment make me think that it's better to have this information in the repo. But one thing we can do is see where most of the questions originate. :-)

Reply:
I think they are both important. Yes, if you have access to both, there's redundancy. And yes, it is harder to keep two documentations up to date. However, confluence has restricted access, so I'd like to keep some documentation in the repo (which is public). Not that we are very interested in users support outside of E3SM ecosystem, but it feels "right". Besides, Jeff is right, that documentation should be as close as possible to the code. I think a new user is more likely to browse the doc folder than navigating the (big) confluence space.